aptos-move/mvhashmap/src/lib.rs

```diff
@@ -71,6 +71,12 @@ impl<
         &self.group_data
     }
 
+    /// Contains data representing resource groups, or more generically, internally
+    /// containing different values mapped to tags of type T.
+    pub fn group_data_mut(&mut self) -> &mut VersionedGroupData<K, T, V> {
```

Comment:
why? i don't see a use in this PR, and in general shouldn't be needed (interior mutability)

Reply:
Artifact of my previous design. Removed it.